crates/relayer/src/transfer.rs

```diff
@@ -195,21 +195,29 @@ fn build_transfer_message_astria(
     timeout_height: TimeoutHeight,
     timeout_timestamp: Timestamp,
 ) -> Any {
-    use astria_core::sequencer::v1::asset::default_native_asset_id;
+    use astria_core::primitive::v1::Address;
+    let sender: Address = sender
+        .to_string()
```

Comment:
This has `impl AsRef<str> for Sender`, so this should work without allocation: `sender.as_ref().parse()`, or more explicitly `AsRef::<str>::as_ref(&sender).parse()` (might be necessary if it can't decide what to use as the as-ref target).